src/urllib3/util/url.py

```diff
@@ -1,20 +1,18 @@
 import re
-from collections import namedtuple
+import typing
```

Comment:
Can we be consistent between `import typing` and `from typing import *`? I slightly prefer the latter.